R/create_table_patient_data_changes_only.R

```diff
@@ -71,7 +71,7 @@ create_table_longitudinal_data <-
             ungroup() %>%
             arrange(id, tracker_year, tracker_month)
 
-        export_data(
+        export_data_parquet(
```

Comment:
just a minor thing but I like names that read properly, so I would like to read `export_data_to|as_parquet`